monkey/infection_monkey/monkey.spec

```diff
@@ -67,6 +67,8 @@ def get_hidden_imports():
     # `cryptography.hazmat.primitives.padding`, pyinstaller will not include it unless we
     # explicitly tell it to. Once the remainder of the exploiters (SSH and Log4Shell) are migrated
     # to plugins, we can attempt to remove the cryptography dependency from the agent entirely.
```

Comment:
We can probably remove this section of the comment because we tried and didn't worked

Reply:
I think we should leave it in for context. If we ever come back to this again, we know what we did and why we're doing what we're doing.